resources/scripts/components/server/files/FileManagerContainer.tsx

```diff
@@ -72,11 +89,14 @@ export default () => {
                         }
                     />
                 </ErrorBoundary>
+
+                <input onChange={searchFiles} css={tw`w-full rounded-lg bg-neutral-700 border-2 border-cyan-700 mx-10 px-4`} placeholder='search' style={{borderColor: 'rgb(59 130 246);'}}></input>
```

Comment:
Use the Field component

Reply:
I tried using the field component in an earlier commit and the CSS spacing got all weird. When I tried setting any CSS elements on the field to fix it the input stopped working.